acoustid/web/views/apps.py

```
@@ -23,36 +24,40 @@
-        """
+        sql.text(
+            """
         SELECT
             date_trunc('month', date) AS month,
             sum(count_hits) + sum(count_nohits) AS lookups
         FROM stats_lookups
         WHERE application_id = %s
         GROUP BY date_trunc('month', date)
         ORDER BY date_trunc('month', date) DESC
-    """,
+    """
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)